constraints/src/main/java/gov/nasa/jpl/aerie/constraints/tree/ActivityWindow.java

```diff
@@ -3,26 +3,24 @@
 import gov.nasa.jpl.aerie.constraints.model.EvaluationEnvironment;
 import gov.nasa.jpl.aerie.constraints.model.SimulationResults;
 import gov.nasa.jpl.aerie.constraints.time.Interval;
-import gov.nasa.jpl.aerie.constraints.time.Segment;
-import gov.nasa.jpl.aerie.constraints.time.Windows;
+import gov.nasa.jpl.aerie.constraints.time.Spans;
+import org.apache.commons.lang3.tuple.Pair;
 
 import java.util.Objects;
+import java.util.Optional;
 import java.util.Set;
 
-public final class ActivityWindow implements Expression<Windows> {
+public final class ActivityWindow implements Expression<Spans> {
```

Comment:
Please rename to ActivitySpan

Reply:
Revised: better for now to have both ActivityWindow and ActivitySpan

Reply:
Yes.